### PR TITLE
k8s-1-13-release

### PR DESCRIPTION
**Pull request by calvix**
also updating etcd version and calico

for https://github.com/giantswarm/giantswarm/issues/4875

### CODE REVIEW

**Comment by teemow** [comment]
If possible we should upgrade calico too. Anything else that is still in here?

**Comment by calvix** [comment]
calico updated too

**Comment by calvix** [comment]
tested on AWS, works like a charm

**Comment by calvix** [comment]
RFR

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ version directory, and then changes are introduced.
 ## [v4.1.0] WIP
 ### Changed
 - Intall calicoctl, crictl and configure etcctl tooling in masters.
+- Update kubernetes to 1.13.3.
+- Update etcd to 3.3.12.
+- Update calico to 3.5.3.
```
Comment (xh3b4sd) [inline CHANGELOG.md]: Below it says `calico/cni:v3.5.1`.

**Comment by corest** [comment]
I had to change this on early 1.13 tests 
- https://github.com/giantswarm/k8scloudconfig/pull/477/files#diff-12742d998dddca1aaec57756e978f22bR1
- https://github.com/giantswarm/k8scloudconfig/pull/477/files#diff-17bf99a6ffc9eed94e9375c748c760e8R54
- https://github.com/giantswarm/k8scloudconfig/pull/477/files#diff-71f0096de78a2da6b9586eeb988ba524R305

File: CHANGELOG.md (hunk 1 above)
Comment (calvix) [reply CHANGELOG.md]: Thanks, somehow missed that

**Comment by calvix** [comment]
integrated the changes from Vol PR

**Comment by calvix** [comment]
another RFR

File: v_4_1_0/files/k8s-resource/calico-all.yaml [hunk 2]
```diff
@@ -393,43 +408,47 @@ spec:
...
 ---
 
-# Calico Version v3.2.3
-# https://docs.projectcalico.org/v3.2/releases#v3.2.3
----
+# Include a clusterrole for the kube-controllers component,
+# and bind it to the calico-kube-controllers serviceaccount.
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (xh3b4sd) [inline v_4_1_0/files/k8s-resource/calico-all.yaml]: Why do we go back in version? That feels wrong. Below it is still `v1`.

Comment (calvix) [reply v_4_1_0/files/k8s-resource/calico-all.yaml]: bah, for some reason the official calico manifest have the old api version :(

File: v_4_1_0/files/k8s-resource/calico-all.yaml [hunk 3]
```diff
@@ -131,12 +131,57 @@ spec:
       # Minimize downtime during a rolling upgrade or deletion; tell Kubernetes to do a "force
       # deletion": https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods.
       terminationGracePeriodSeconds: 0
+      initContainers:
+        # This container installs the Calico CNI binaries
+        # and CNI network config file on each node.
+        - name: install-cni
+          image: calico/cni:v3.5.1
```
Comment (kopiczko) [inline v_4_1_0/files/k8s-resource/calico-all.yaml]: Shouldn't we use retagged image from parametrized registry?

**Pull request merged**